lib/wibox/widget/imagebox.lua

```diff
@@ -135,11 +135,24 @@ local function load_and_apply(ib, file, image_loader, image_setter)
     return image_applied
 end
 
---- Set an imagebox' image
+--- The image rendered by the `imagebox`.
+-- <br />It can can be any of the following:
+--
+-- * A `string` : Interpreted as the path to an image file ;
+-- * A cairo image surface : Directly used as is ;
+-- * An rsvg handle object : Directly used as is ;
+-- * `nil` : Deny previously setted image.
```

Comment:
`Deny previously setted image` sounds weird to me. How about `Unset image`? `Remove image`? Dunno...

Reply:
It was the words used before. You're right, tho. I'll change that